sql/sql_reload.cc

```diff
@@ -516,6 +517,14 @@ bool flush_tables_with_read_lock(THD *thd, TABLE_LIST *all_tables)
   Lock_tables_prelocking_strategy lock_tables_prelocking_strategy;
   TABLE_LIST *table_list;
 
+  /*
```

Comment:
Move this a few lines down, before:
  if (thd->current_backup_stage != BACKUP_FINISHED)
  {
    my_error(ER_BACKUP_LOCK_IS_ACTIVE, MYF(0));
    return true;
  }

Same reason as previous comment:  Code is easier to understand if all backup related tests are close to each other

Reply:
Moved this one too, as suggested.